RelayProofsOfConcept/EddsaTendermintServer/relay-server-common/src/lib.rs

```diff
@@ -146,6 +146,76 @@ impl StoredMessages {
             _ => (),
         }
     }
+
+    pub fn get_number_messages(&self, round: &u32) -> usize {
+        match self.messages.get(round) {
+            Some(messages) => return messages.keys().len(),
+            None => 0,
+        }
+    }
+
+    // Returns the messages of the current round as relay message format,
+    // or an empty vector if no messages are stored for the round
+    pub fn get_messages_vector_relay_message(&self, round: &u32) -> Vec<RelayMessage> {
```

Comment:
why do we sometime borrow `round` (in `get_messages_vector_relay_message` & `get_messages_vector_client_message`, `get_number_messages`...), but sometime don't (`get_missing_clients_vector`)

Reply:
Good point, will fix to be consistent